CMakeLists.txt

```diff
@@ -42,6 +45,8 @@ endif()
 #  "${CMAKE_CXX_FLAGS} -municode"
 #)
 
+set(CMAKE_BUILD_TYPE Debug)
+
```

Comment:
NO THIS IS SET THOUGH THE BUILD COMMAND DON'T DO THIS

Reply:
lol I forgot I did this. oops